stock_barcode_alternative/i18n/sv.po

```diff
@@ -0,0 +1,135 @@
+# Translation of Odoo Server.
+# This file contains the translation of the following modules:
+#	* stock_barcode_alternative
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: Odoo Server 8.0\n"
```

Comment:
Maybe change to version to 14.0? If no changes has been made for Odoo 14 it is valid to leave at 8.0 to highlight the files history.

Reply:
Alright, I must have missed that. I will fix